### DISCUSSION TITLE
container not starting up

### DISCUSSION THREAD

**Discussion by klmbrgr** [qdm12/gluetun#1822] [Q&A] [answered]
Hi there,
I want to run a surfshark connection along with Jdl2. Therefore I have created a stack as described in [HERE](https://code.mendhak.com/run-docker-through-vpn-container/).
The service stays unhealthy, the log is giving: 

> gluetun-gluetun-1  | 2023-08-28T19:47:46Z INFO [firewall] allowing VPN connection...
> gluetun-gluetun-1  | 2023-08-28T19:47:46Z INFO [wireguard] Using available kernelspace implementation
> gluetun-gluetun-1  | 2023-08-28T19:47:46Z INFO [wireguard] Connecting to 89.37.173.47:51820
> gluetun-gluetun-1  | 2023-08-28T19:47:46Z INFO [wireguard] Wireguard setup is complete. Note Wireguard is a silent protocol and it may or may not work, without giving any error message. Typically i/o timeout errors indicate the Wireguard connection is not working.
> gluetun-gluetun-1  | 2023-08-28T19:49:02Z INFO [healthcheck] program has been unhealthy for 1m16s: restarting VPN (see https://github.com/qdm12/gluetun-wiki/blob/main/faq/healthcheck.md)
> gluetun-gluetun-1  | 2023-08-28T19:49:02Z INFO [vpn] stopping
> gluetun-gluetun-1  | 2023-08-28T19:49:02Z INFO [vpn] starting

Any help pushing to the right direction is very appreciated.

Thanks!

**Comment by klmbrgr** [comment, marked as answer]
If it helps anyone - I found the problem and the solution.
The fault was the surfshark configuration . After downloading the config from the surfshard and opening it with notepad and copying the values from there, the container started up successfully! ;)

**Reply by Marlow007** [reply]
@kolmberger I have a similar issue but with NordVPN. What config file did you download? are you using an Unraid Docker Configuration?

**Reply by klmbrgr** [reply]
@Marlow007 no, my setup is on a debian system (openmediavault) with docker compose. See the linked description for surfshark. First, create a VPN config on your provider side, download the config file and work with the parameters in it.
Hope it helps.
